webrtc.html

```diff
@@ -5945,10 +5945,24 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       <div>
         <pre class="idl">dictionary RTCRtpCodecCapability {
              DOMString mimeType;
+             unsigned long  clockRate;
+             unsigned short channels = 1;
```

Comment:
Oh, right. Not an issue if the application just takes capabilities and reorders them, but it might create the dictionaries from scratch.